### ISSUE TITLE
Replace histograms for numerical centered data in enRoute

### ISSUE THREAD

**Issue by alexsb** [Caleydo/caleydo#1714]
Right now we use the histos of the categorical data. This doesn't make too much sense. Instead we should use a visual encoding similar to the not-centered data, where we currently use bars & error bars. 

I think we could consider replacing bars & error bars both with this representation:

http://graphics.cs.wisc.edu/Papers/2013/CG13/

Opinions?

**Comment by cpartl** [comment]
RFC. Currently box plots are used, but simply replace instantiations of `SummaryBoxAndWhiskersRenderer` with `SummaryGradientPlotRenderer` to see the gradient plots.

**Comment by alexsb** [comment]
Here is a comparison: 

![box](https://f.cloud.github.com/assets/2039375/1544532/9746b392-4d68-11e3-8b00-e53bbe04a15a.png)
![gradient](https://f.cloud.github.com/assets/2039375/1544535/9cbaef96-4d68-11e3-8d26-4bda1f3785d9.png)

**Comment by alexsb** [comment]
Overall, I like it a lot!  Both versions. 

Some comments:

There seems to be some imprecision in placing the center line:

![box_detail](https://f.cloud.github.com/assets/2039375/1544592/1bf073f8-4d69-11e3-8a6b-d376acae3abb.png)

Some other suggestions: 
- make the median line white. 
- make the 0-axis gray?
- alternative: just hint at the 0-axis with tics?

**Comment by alexsb** [comment]
Love the tool-tip :+1: Maybe add std deviation? Can you break lines in tool tips? Would make it easier to parse.

**Comment by cpartl** [comment]
@alexsb I tried the tics, but they cannot perceived as center line any more imo. 
I did not want to make the center line gray for 2 reasons: Contrast to background and uniformness with detailed centered data rendering.
A white median does not look that great but I guess it's also a matter of taste.
I could add a std dev to the tooltip, but I wanted to keep the tooltip to what we show, and I guess it would potentially confuse the user with respect to what the whiskers mean.
I have now added a very simplistic design without color, as it does not mean anything anyway. RFC.
![box_style](https://f.cloud.github.com/assets/4134949/1549083/6333894e-4ddc-11e3-9f81-8af6a8ea0d5f.png)
The problem with this though is that white probably won't work for gradient plots.
